tests/admin_views/tests.py

```diff
@@ -2736,7 +2736,7 @@ def test_list_editable_pagination(self):
         self.assertContains(response, 'Unordered object #3')
         self.assertContains(response, 'Unordered object #2')
         self.assertNotContains(response, 'Unordered object #1')
-        response = self.client.get(reverse('admin:admin_views_unorderedobject_changelist') + '?p=1')
+        response = self.client.get(reverse('admin:admin_views_unorderedobject_changelist') + '?p=2')
```

Comment:
Hm, if you mean, that previous links to admin changelist pages would show another results after proposed change - yes, that's correct.

I raised that concern previously - https://github.com/django/django/pull/5866#issuecomment-168448650.

The reason of the case is simple - pagination page parameter is one less that actual page number, e.g. `reverse('admin:admin_views_unorderedobject_changelist') + '?p=2'` is showing 3rd page.